### PR TITLE
fix(docs): update how to add/enable a new language

### LINKED ISSUE #48430: Update Docs to describe new structure of languages and superblocks
### Describe the Issue

[This PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/47596) was merged and it changed the way the languages and superblocks are configured in the codebase. The documentation needs to be updated still to reflect the new changes.

### Affected Page

https://contribute.freecodecamp.org/

### PR DESCRIPTION
**Pull request by moT01**
This updates the docs to describe the new structure and steps after refactoring the maps (superblock order) on [this PR.](https://github.com/freeCodeCamp/freeCodeCamp/pull/47596/files)

The two main files changed on that PR are `config/superblock-order.ts` and `config/i18n.ts`.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48430

### CODE REVIEW

File: docs/how-to-enable-new-languages.md [hunk 1]
```diff
@@ -145,6 +120,72 @@ export const rtlLangs = [''];
...
+        [SuperBlockStates.New]: [],
+        [SuperBlockStates.Upcoming]: [],
+        [SuperBlockStates.Legacy]: []
+      }
+    }
+  }
+}
+```
+
+The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Follow the comments in that file so you know how you are allowed to order the superblocks, then move them to their proper places for the new language.
```
Comment (Sboonny) [inline docs/how-to-enable-new-languages.md:175]:
```suggestion
The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Follow the comments in that file to know how you can order the superblocks, then move them to their proper places for the new language.
```

---

It feels weird for me to tell someone who is reading the docs, "follow the comments in the file" while they are already in the docs.

```suggestion
The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Move the superblocks in the new language's arrays, which positions them in their proper places.
```

File: docs/how-to-enable-new-languages.md [hunk 2]
```diff
@@ -145,6 +120,72 @@ export const rtlLangs = [''];
...
+
+> [!ATTENTION]
+> Do not change the order of any of the keys in the object, just move the superblocks to the different arrays
+
+The `CurriculumMaps.Landing` array should contain exactly one superblock for all our current certifications; and the `CurriculumMaps.Learn` object should have all existing superblocks in it. Translated superblocks go in `TranslationStates.Audited` and non-translated superblocks go in `TranslationStates.NotAudited`. Each of those two objects has four different states a superblock can be in.
+
+- `SuperBlockStates.Current`: means that the superblock is current. e.g. `(New) Responsive Web Design`
```
Comment (Sboonny) [inline docs/how-to-enable-new-languages.md:182]: silly question, we don't apply the writing style for the challenges in the docs, too. 

Whelp, here is a suggestion if we do apply the writing style.

```suggestion
- `SuperBlockStates.Current`: means that the superblock is current, example: `(New) Responsive Web Design`.
```

File: docs/how-to-enable-new-languages.md [hunk 3]
```diff
@@ -145,6 +120,72 @@ export const rtlLangs = [''];
...
+  }
+}
+```
+
+The order of the superblocks in this object is how they appear on the "Landing" page and "Learn" maps. Follow the comments in that file so you know how you are allowed to order the superblocks, then move them to their proper places for the new language.
+
+> [!ATTENTION]
+> Do not change the order of any of the keys in the object, just move the superblocks to the different arrays
+
+The `CurriculumMaps.Landing` array should contain exactly one superblock for all our current certifications; and the `CurriculumMaps.Learn` object should have all existing superblocks in it. Translated superblocks go in `TranslationStates.Audited` and non-translated superblocks go in `TranslationStates.NotAudited`. Each of those two objects has four different states a superblock can be in.
```
Comment (Sboonny) [inline docs/how-to-enable-new-languages.md:180]:
```suggestion
The `CurriculumMaps.Landing` array should contain exactly one superblock for all our current certifications, and the `CurriculumMaps.Learn` object should have all existing superblocks in it. Translated superblocks go in `TranslationStates.Audited` and non-translated superblocks go in `TranslationStates.NotAudited`. Each of those two objects has four different states a superblock can be in.
```

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

Should have asked if this ready for review first 😅

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**